v2/cli/term/project_page.go

```diff
@@ -0,0 +1,328 @@
+package term
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/brigadecore/brigade/sdk/v2/meta"
+	"github.com/gdamore/tcell/v2"
+	"github.com/rivo/tview"
+	"k8s.io/apimachinery/pkg/util/duration"
+)
+
+const projectPageName = "project"
+
+// projectPage is a custom UI component that displays Project info and a list of
+// associated Events.
+type projectPage struct {
+	*page
+	currentProjectID     string
+	projectInfo          *tview.TextView
+	eventsContinueValues []string // Stack of "continue" values to aid paging
+	eventsTable          *tview.Table
+	usage                *tview.TextView
+}
+
+// newProjectPage returns a custom UI component that displays Project info and a
+// list of associated Events.
+func newProjectPage(
+	apiClient core.APIClient,
+	app *tview.Application,
+	router *pageRouter,
+) *projectPage {
+	p := &projectPage{
+		page:                 newPage(apiClient, app, router),
+		projectInfo:          tview.NewTextView().SetDynamicColors(true),
+		eventsContinueValues: []string{""}, // "" == continue value for first page
+		eventsTable:          tview.NewTable().SetSelectable(true, false),
+		usage:                tview.NewTextView().SetDynamicColors(true),
+	}
+	p.projectInfo.SetBorder(true).SetBorderColor(tcell.ColorWhite)
+	p.eventsTable.SetBorder(true).SetTitle(" Events ")
+	// Create the layout
+	p.page.Flex = tview.NewFlex().
+		SetDirection(tview.FlexRow).
+		AddItem(p.projectInfo, 0, 2, false).
```

Comment:
It may be helpful to add a comment explaining arguments 2-4 for the `AddItem` function somewhere...